styles.css

```diff
@@ -0,0 +1,211 @@
+@import url('https://fonts.cdnfonts.com/css/poppins');
+
+:root {
+  --main-bg-color: #e5e5e5;
+  --menu-bg-color: #6070ff;
+  --header-text-color: #172b4d;
+  --text-color: #344563;
+  --icon-color: #505f79;
+  --primary-color: #6070ff;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: var(--main-bg-color);
+}
+
+header {
+  display: flex;
+  align-items: center;
+  padding: 10px 24px;
+  justify-content: space-between;
+  background-color: white;
+  position: fixed;
+  width: 100%;
+}
+
+main {
+  overflow-y: auto;
+}
+
+.logo {
+  font-family: 'Poppins', sans-serif;
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  text-align: center;
+  color: var(--primary-color) !important;
+}
+
+.bio {
+  height: 718px;
+  width: 100%;
+  background-color: #fff;
+  background-image: url('./assets/images/header-bg.svg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 150px 24px;
+  background-origin: border-box;
+  border-bottom-left-radius: 20%;
+}
```

Comment:
- [ ] as mentioned by the previous reviewer the background position still doesn't match the Figma design , kindly fix it by updating your code to the snippet below : 

``` CSS
.bio {
    height: 718px;
    width: 100%;
    background-color: #fff;
    background-image: url(./assets/images/header-bg.svg);
    background-repeat: no-repeat;
    /* set background position to center */
    background-position: center;
    /* change background size to contain*/
    background-size: contain;
    padding: 150px 24px;
    background-origin: border-box;
    border-bottom-left-radius: 20%;
}
```